src/app/src/css/quiz.css

```
@@ -0,0 +1,18 @@
+.quiz {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.quiz__description {
+    width: 500px;
+    text-align: center;
+    margin-bottom: 50px;
+}
+
+.quiz__button {
+    border-radius: 15px 15px 15px 15px;
```

This can be simplified to:
```suggestion
    border-radius: 15px;
```

---

🤦‍♀